supabase/migrations/20260422_add_adaptive_race_engineer.sql

```
@@ -0,0 +1,526 @@
+    )
+  );
+
+create policy "session_environment: delete own"
+  on public.session_environment for delete
+  using (auth.uid() = user_id);
+
+create policy "ai_recommendations: select own"
+  on public.ai_recommendations for select
+  using (auth.uid() = user_id);
+
+create policy "ai_recommendations: insert own"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)